### ISSUE TITLE
OddNews Issue 10

### ISSUE THREAD

**Issue by SondraE** [oddbird/oddleventy#528]
**Issue Number**: 10
**Publish Date**: Mar 27 or 28, 2024
**Link to Archive**: [OddNews Issue 10](https://us19.campaign-archive.com/?e=[UNIQID]&u=80219aa68d7bad77b9fd2eb93&id=949bbdde43)

## Create the Issue
- [x] Review last issue's analytics for most clicked content
- [x] Gather content from Birds
- [ ] Create Summary of content
- [ ] Create Misc Tool/Announcement/Event/News
- [ ] Create Article Spotlight
- [ ] Create Bird of the Month
- [ ] Review @mirisuzanne 
- [ ] Review @stacyk 
- [ ] Review @jamesnw 
- [ ] Review @jgerigmeyer 
- [ ] Revise @SondraE 
- [x] Schedule publish on Wed ro Thu around 11a MT

## Marketing Checklist
- [ ] Add issue to OddSite archive (via MailChimp admin)
- [ ] Add archive link to Trello card
- [ ] Promote on Mastodon
- [ ] Promote on LinkedIn
- [ ] Promote on Twitter

**Comment by SondraE** [comment]
@stacyk Last time we published a newsletter, you suggested that we feature a CSS bird this time. 
Do you have suggestions? 
How about something like this? https://codepen.io/YusukeNakaya/pen/BwBgvq
Or should we make our own?

**Comment by jamesnw** [comment]
@SondraE The Winging it link is incorrect- it should be https://www.oddbird.net/2024/03/21/winging-it-05/ (it currently is 3/19).

**Comment by dvdherron** [comment]
> @mirisuzanne @dvdherron Last time we published a newsletter, Stacy suggested that we feature a CSS bird this time. Do you all have suggestions? How about something like this? https://codepen.io/YusukeNakaya/pen/BwBgvq Or should we make our own? (I would have asked Stacy, but she's out for the week at this point.)

I think that's a pretty cool bird! I'm cool with using this. Reminded me of [my variable font bird thing](https://variable-birdsong.netlify.app/). I probably need to clean that up and we can use it in another newsletter!

**Comment by jgerigmeyer** [comment]
I fixed the Winging It URL and added in "Sass" to both the subject and first paragraph when referencing package imports -- @SondraE feel free to tweak the subject (should "a" be capitalized?), I just think "Node.js package importer" doesn't make sense without referencing Sass.

**Comment by SondraE** [comment]
@dvdherron 
> Reminded me of [my variable font bird thing](https://variable-birdsong.netlify.app/).

Yes! This is so cool! If you have a chance to clean that up in the next 4 or 5 weeks, we can use it in the next newsletter!

**Comment by mirisuzanne** [comment]
@SondraE `Alvoro Montoro` should be `Alvaro Montoro` - otherwise lgtm

**Comment by jgerigmeyer** [comment]
> @SondraE `Alvoro Montoro` should be `Alvaro Montoro` - otherwise lgtm

I was in there fixing another typo, so I fixed this as well

**Comment by jerivas** [comment]
LGTM 👍
